source/blog/2021-04-19-dbal-2.13.1-3.1.0.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "New Release: Doctrine DBAL 2.13.1 and 3.1.0 with important Forward Compatibility fix"
+authorName: Benjamin Eberlei
+authorEmail: kontakt@beberlei.de
+permalink: /2021/04/19/dbal-2.13.1-3.1.0.html
+---
+
+Last month [we released DBAL
+2.13.0](https://www.doctrine-project.org/2021/03/29/dbal-2.13.html) as an
+important push for the ecosystem towards DBAL 3 with an extensive deprecation
+layer and forwards compatibility.
```

Comment:
Below, you used "forward", and I think you should pick that one ("forward"). Also, should it read "with an extensive deprecation and forward compatibility layer"?